brushfire-finatra/build.sbt

```diff
@@ -0,0 +1,20 @@
+name := "brushfire-finatra"
+
+resolvers += Resolvers.twttr
```

Comment:
Hrmm... When I was first building, one of the deps seemed to not be on maven central, but now that I rebuild (with that line commented out) that problems seems to have disappeared. Not sure. I'll remove it now!